.github/workflows/docstring_validation.yml

```diff
@@ -0,0 +1,50 @@
+name: Test Docstrings Validation
+
+on:
+  pull_request:
+    paths:
+      - "integration-tests/**"
+
+jobs:
+  betelgeuse:
+    name: "betelgeuse dry-run"
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+
+    - uses: actions/checkout@v4
+
+    steps:
+    - name: Base setup for Betelgeuse
+      run: |
+        dnf --setopt install_weak_deps=False install -y \
+            python3-pip
+        python3 -m pip install betelgeuse
+        cp integration-tests/custom_betelgeuse_config.py \
+          /usr/local/lib/python3.*site-packages/
```

Comment:
ugh, this is ugly, and also requires a read-write python installation to test locally

i'll add a separate comment for a working way to use the local config as-is